### PR TITLE
feat: support portal

### PR DESCRIPTION
**Pull request by japhethLG**
# Description

Improved the UI and functionality of the ticket viewing section in the support portal. The updates include fixing the display of messages, images, and attachments, adding date and time displays for messages, ensuring only the agent's name is displayed per message, and enhancing attachment visibility for image and other file types.  Implemented a dark mode compatible UI, introduced an information bar on the right side, and refactored the code for fetching threads and attachments to improve the loading experience. A regex fix was added to handle threads without content.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Conducted manual testing by creating a ticket and using the Zoho portal to send and receive an email. This process was used to verify the correct display of messages, images, attachments, dates, and times, as well as the functionality of the new features.

- [x] Manual Test

# Screenshots / Screen recording

Screenshot 2024-02-06 154906
Screenshot 2024-02-06 155009
image viewer
dark mode

### CODE REVIEW

File: src/revamp/ui/GetDemoSection/index.js
Comment (giseleblair) [inline src/revamp/ui/GetDemoSection/index.js:1]: why are there changes to this component?

**Review by giseleblair — CHANGES_REQUESTED** [review]
please review why there are edits to the GetDemoSection component, is it just prettier running its formatting or were there changes to the function?

File: src/revamp/ui/GetDemoSection/index.js
Comment (japhethLG) [reply src/revamp/ui/GetDemoSection/index.js:1]: I'm not sure why Jomar and Darwin's commit was mixed in here, will investigate this.

Comment (japhethLG) [reply src/revamp/ui/GetDemoSection/index.js:1]: Found the issue, I rebased this branch to not include the unrelated commits.

**Comment by giseleblair** [comment]
Screenshot 2024-02-07 at 10 51 13 AM

@japhethLG can you please review the above message

**Comment by japhethLG** [comment]
> Screenshot 2024-02-07 at 10 51 13 AM
> 
> @japhethLG can you please review the above message

Merged stage to this branch.

**Pull request merged**